frontend/packages/kubevirt-plugin/src/selectors/pod/selectors.ts

```diff
@@ -46,40 +39,6 @@ export const isPodSchedulable = (pod: PodKind) => {
   );
 };
 
-export const findVMPod = (
```

Comment:
what is wrong with this function? Why can't we keep it ?

Reply:
switched back :-)